css/styles.css

```diff
@@ -386,3 +388,100 @@ button.form_btn:hover {
   grid-template-rows: 25% 25% 25% 25%;
   row-gap: 20px;
 }
+
+.contenido-form {
+  display: flex;
+}
+
+/*      Desktop Style     */
+
+@media (min-width: 768px) {
+  .portfolio {
+    display: flex;
+    display: none;
+  }
+
+  .portfolio1 {
+    display: flex;
+    width: auto;
+  }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .project-detail {
+    width: 569px;
```

Comment:
- [ ] I think it's better to avoid the use of fixed width in your style. Prefer the use of % that could be adapted for any size of the screen.
Your app:
<img width="580" alt="Screen Shot 2022-08-27 at 12 13 24 AM" src="https://user-images.githubusercontent.com/47673311/187012260-ebcb43e2-3fb9-4d6e-8126-0fcc52b56e86.png">

- [ ] Please correct the space between cards following the Figma template. It's a project requirement.
- [ ] Please stick to the Figma template and change the position of the image for each card. You can use the [nth-of-type odd or even](https://www.w3schools.com/cssref/sel_nth-of-type.asp) with `order` in Grid or Flexbox.